docs/send-data/kubernetes/v4/important-changes.md

```diff
@@ -45,8 +45,28 @@ Ensure that the following CRDs from the OpenTelemetry operator are installed and
 Please follow instructions below to install the appropriate CRD versions
 :::
 
-#### CRDs to install (v4.12.0 and later)
 
+#### CRDs to install (v4.16.0 and later)
+
+```shell
+kubectl apply --server-side -f https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/refs/tags/v4.12.0/deploy/helm/sumologic/crds/crd-opentelemetry.io_opampbridges.yaml --force-conflicts
```

Comment:
```suggestion
kubectl apply --server-side -f https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/refs/tags/v4.16.0/deploy/helm/sumologic/crds/crd-opentelemetry.io_opampbridges.yaml --force-conflicts
```

Same for the other kubectl commands

Reply:
We could also remove the note here - https://help.sumologic.com/docs/send-data/kubernetes/install-helm-chart/#helm